app/src/main/java/com/ncnf/repositories/UsersRepository.java

```diff
@@ -0,0 +1,24 @@
+package com.ncnf.repositories;
+
+import com.ncnf.database.DatabaseService;
+import com.ncnf.user.User;
+
+import java.util.concurrent.CompletableFuture;
+
+import javax.inject.Inject;
+
+import static com.ncnf.utilities.StringCodes.USERS_COLLECTION_KEY;
+
+
+public class UsersRepository {
+
+    @Inject
+    public DatabaseService db;
+
+    @Inject
+    public UsersRepository(){}
```

Comment:
Is this constructor necessary ?

Reply:
Changed that to match FriendsRepository ! This is no longer the case !